src/modules/chat_in_fullscreen/index.js

```diff
@@ -0,0 +1,44 @@
+const $ = require('jquery');
+const settings = require('../../settings');
+
+class ChatInFullscreen {
+    constructor() {
+        settings.add({
```

Comment:
rather than adding a standard setting, i would prefer a player button when a user is in fullscreen